assets/large_language_models/rag/environments/rag_embeddings/context/conda_dependencies.yaml

```diff
@@ -6,22 +6,53 @@ dependencies:
 - python=3.8
 - pip=21.3.1
 - pip:
+    # RAG package
   - azureml-rag[azure,faiss,cognitive_search,document_parsing,data_generation,pinecone]>=0.2.18.1
-  - langchain==0.0.317
+    # Azure AI
+  - azure-ai-formrecognizer==3.3.1
+  - azure-ai-ml=={{latest-pypi-version}}
+    # Azure
+  - azure-core<2.0.0,>=1.8.0,!=1.22.0
+  - azure-identity==1.12.0
+  - azure-keyvault-secrets==4.6.0
+  - azure-mgmt-cognitiveservices~=13.4.0
+  - azure-mgmt-core<2.0.0,>=1.3.0
+  - azure-search-documents==11.4.0b8
+    # Azure ML
+  - azureml-contrib-services
   - azureml-core=={{latest-pypi-version}}
-  - azureml-telemetry=={{latest-pypi-version}}
+  - azureml-dataprep>=4.12.7 # Change back to latest version once azureml-fsspec is compatible
+  - azureml-fsspec
+  - azureml-inference-server-http==0.7.6
   - azureml-mlflow=={{latest-pypi-version}}
-  - azure-search-documents==11.4.0b8
-  - psutil~=5.8.0
-  - pandas~=1.1.5
+  - azureml-telemetry=={{latest-pypi-version}}
+    # Other public packages
+  - beautifulsoup4~=4.11.2
+  - datasets~=2.10.1
+  - faiss-cpu~=1.7.3
+  - fsspec~=2023.3.0
   - GitPython>=3.1
+  - inference-schema==1.4
+  - langchain==0.0.317
+  - lxml
+  - markdown
+  - mlflow>=2.6.0
+  - mlflow-skinny==2.3.2
+  - msal~=1.22.0
   - msrest>=0.6.18
-  - azure-core<2.0.0,>=1.8.0,!=1.22.0
-  - azure-mgmt-core<2.0.0,>=1.3.0
-  - azure-keyvault-secrets==4.6.0
-    # package dependency needed for aml run token authentication
+  - nltk==3.8.1
+  - openai~=0.27.4
+  - pandas~=1.1.5
+  - polling2~=0.5.0
+  - psutil~=5.8.0
+  - pymssql==2.2.7
+  - pypdf~=3.7.0
+  - sentence-transformers
+  - sqlalchemy==1.4.46
+  - tika~=2.6.0
+  - tiktoken~=0.3.0
```

Comment:
I don't think we need this, azureml-rag has a specific verson range that's higher

Reply:
azureml-rag defined following DEPENDENCIES: 
    "azureml-dataprep[parquet]>4.11",
    "azureml-fsspec",
    "fsspec~=2023.3",
    "openai>=0.27.8,<1",
    "tiktoken<0.6",
    "cloudpickle",
    "mmh3",
    "pyyaml>=5.1.0,<7.0.0",
    "requests",

Reply:
"azureml-fsspec","fsspec~=2023.3","tiktoken~=0.3.0" are removed in next iteration.